config/initializers/devise.rb

```diff
@@ -0,0 +1,289 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2008-2013 Michael Dvorkin and contributors.
+#
+# Fat Free CRM is freely distributable under the terms of MIT license.
+# See MIT-LICENSE file or http://www.opensource.org/licenses/mit-license.php
+#------------------------------------------------------------------------------
+require 'devise-encryptable'
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
```

Comment:
Do we need to make it so this is easier to override/customise/etc? Thinking vaguely of things like
 FFCRM used in another project where there is devise configuration already; and we accidentally lower security or break their authentication 

We sort of do that with ```config/settings.default.yml``` and allowing users to override that with their custom versions; would be interesting if we could do the same here.

Reply:
Neither Devise nor FFCRM play nicely being used a part of other projects, so I don't think this is necessary. If someone wants to do it (which I expect is unlikely) it can be a separate PR.